project/src/main/java/com/google/sps/servlets/ServletExampleWithInjection.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import com.google.sps.utilities.DatabaseWrapper;
+import com.google.sps.utilities.InstanceDatabaseWrapper;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns the login status of the user. */
+@WebServlet("/it-servlet-example")
+public class ServletExampleWithInjection extends HttpServlet {
```

Comment:
maybe name class better than 'ServletExampleWithInjection' to what the Servlet actually does, and comment that this is the injection example... same with test name